plutus-metatheory/plutus-metatheory.cabal

```diff
@@ -251,197 +260,6 @@ library
     MAlonzo.Code.Untyped.RenamingSubstitution
     MAlonzo.Code.Utils
     MAlonzo.RTE
-    Opts
-    Raw
-    Scoped
-    Untyped
-
-  autogen-modules:
```

Comment:
Are you sure we can remove this? It sounds like we should have to keep it: https://cabal.readthedocs.io/en/stable/cabal-package.html#pkg-field-autogen-modules

Reply:
From the docs: commands like sdist will fail if the autogenerated modules are not listed in `autogen-modules`.
This is irrelevant to us I think? 
Still, I put them back just in case

Reply:
Unclear to me, maybe we want to sdist occasionally :thinking: I think it's good hygiene to retain sdist-ability.